components/calendar/src/chinese.rs

```diff
@@ -273,42 +274,40 @@ impl Calendar for Chinese {
                 12 => tinystr!(4, "M12"),
                 _ => tinystr!(4, "und"), // maximum num of months in a non-leap year is 12
             }
+        } else if ordinal == leap_month {
+            match ordinal {
+                1 => tinystr!(4, "und"), // cannot have a leap month before the actual month
```

Comment:
nit: add assertions for these branches, they should never be hit

probably `debug_assert!(ordinal > 1 && ordinal < 13)`

Reply:
Done